.storybook/preview.scss

```diff
@@ -0,0 +1,7 @@
+@import "../sass/mdn-minimalist";
```

Comment:
This path needs to be updated to `@import "~mdn-minimalist/sass/mdn-minimalist";`

Reply:
Oh I see-- it's copying over the scss from `node_modules`. Is there a reason why we want to do that over importing directly from the package?

Reply:
Oooh, I did not realize it supports that style of importing. I was having problems with file references and that is why I resorted to copying the SASS and typography into the root from the Node module. Perhaps this will resolve that issue.

Will give it a whirl. I mean in the end we would want to be able to `require/import` these things into the base React component and then individual components like `breadcrumbs` for example from the module anyhow. Thanks @mindy

Reply:
It Works! o/\\o